src/Parser/FrostMage/Modules/Items/ShardOfTheExodar.js

```diff
@@ -0,0 +1,43 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import { formatNumber } from 'common/format';
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class ShardOfTheExodar extends Module {
+
+	damage = 0;
+
+  static dependencies = {
+		combatants: Combatants,
+	};
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasFinger(ITEMS.SHARD_OF_THE_EXODAR.id);
+  }
+
+  on_byPlayer_damage(event){
+    const BLOODLUST_BUFFS = [
+      SPELLS.HEROISM.id,
+      SPELLS.BLOODLUST.id,
+      SPELLS.TIME_WARP.id,
+      SPELLS.NETHERWINDS.id,
+      SPELLS.ANCIENT_HYSTERIA.id,
+      SPELLS.DRUMS_OF_FURY.id,
+      SPELLS.DRUMS_OF_RAGE.id,
+      SPELLS.DRUMS_OF_THE_MOUNTAIN.id,
+      ];
+    if (BLOODLUST_BUFFS.some(buff => this.combatants.selected.hasBuff(buff, event.timestamp))) {
+      this.damage += event.amount;
+    }
+  }
+
+  item() {
+    return {
+      item: ITEMS.SHARD_OF_THE_EXODAR,
+      result: `${formatNumber(this.damage)} Total Damage During Lust`,
```

Comment:
Standard format for items when it comes to showing damage is 
```jsx
result: `${this.owner.formatItemDamageDone(this.damage)}`,
```
This shows something like "7,38% / 126k DPS". You're free to add any additional info in the tooltip (look into other modules that use `<dfn>`tag in the item result)